src/managers/notification-manager.ts

```diff
@@ -48,7 +48,13 @@ class NotificationManager extends LitElement {
       toast = this._toast;
     }
     toast.setAttribute("dir", computeRTL(this.hass) ? "rtl" : "ltr");
-    this._action = action || undefined;
+    if (action) {
+      this._action = action;
+    } else {
+      this._action = {
+        text: this.hass.localize("ui.notification_toast.dismiss"),
+      };
```

Comment:
This part should move to the caller of the toast, we should be able to decide per toast if it is dismissable. Like the connection lost toast should not be dismissable

Reply:
We should make it so that the dismiss is optional but on by default no?

I think a toast should usually be dismissable and should only not be dismissable in fewer cases. 

Adding this as the default I think is good.

Reply:
For most toasts, it is not needed as they will be hidden automatically.

Also material guidelines:

![image](https://user-images.githubusercontent.com/5662298/95604442-e710c300-0a57-11eb-8710-f9794cd623e5.png)

Reply:
They will be hidden after X time. But you have to wait form that. Also, just a caution guideline 😉

Reply:
Maybe the onClick was the best idea instead of a button action on each. In the Google Photos App. If I delete an image. I can click the toast to dismiss it. as well as there is an action to undo the action taken.

Reply:
@zsarnett @bramkragten So, should I revert to toast onClick or add a dismiss button on some toasts?

Reply:
Add a dismiss button for some toasts, like the HA is starting one. We should not allow dismissing them all.

Reply:
@bramkragten Ok, is there any more toasts besides that one?

Reply:
Not sure, you can search the code base for `showToast`